xml/net_samba.xml

```diff
@@ -778,13 +730,74 @@
     configuration.
    </para>
   </sect2>
+  <sect2 xml:id="sec.samba.client.old-server">
+   <title>Enabling Clients to Work With SMB1 Servers</title>
+   <para>
+    The first version of the SMB network protocol, SMB1, is an old and
+    insecure protocol which has been deprecated by its originator, Microsoft.
+    For security reasons, &productname; now expects shares to use the
+    protocol version SMB 2.1, 3.0, or 3.02 by default.
+   </para>
+   <note>
+    <title>SMB1 Deprecation Only Affects Clients Using <command>mount</command></title>
+    <para>
+     SMB1 is still supported by the Samba server software shipped with
+     <!-- NB: profiling below is intentional - we don't support using SLED as a
+     Samba server. -->
+     <phrase os="sles;sled">&sls;</phrase><phrase os="osuse">&osuse;</phrase>.
+     SMB1 is also still supported by <command>smbclient</command>.
+    </para>
+   </note>
+   <para>
+    In an existing setup, this can lead to issues in the following cases:
```

Comment:
The question is: 'which issues'? A brief high-level description of the _matter_ of issues to be expected would be helpful IMHO.